src/lib/shadow-shim-helper-rs/build.rs

```diff
@@ -18,7 +18,7 @@ fn run_cbindgen(build_common: &ShadowBuildCommon) {
         })
         .generate()
         .expect("Unable to generate bindings")
-        .write_to_file("./shim_helper.h");
+        .write_to_file("../../../build/src/lib/shadow-shim-helper-rs/shim_helper.h");
```

Comment:
What about putting the c bindings in OUT_DIR as well for consistency, and adding OUT_DIR to the include path for the C code? But either way is fine.

Reply:
Yeah, it'd be nice to do this, but I don't know of a way to add *this* crate's OUT_DIR to the header search path of dependent crates.